src/query.rs

```diff
@@ -1079,6 +1097,23 @@ impl<Q: Query> PreparedQuery<Q> {
 
         unsafe { PreparedQueryIter::new(meta, archetypes, state.iter()) }
     }
+
+    /// Provide random access to query results for a uniquely borrow world
+    pub fn query_map_mut<'q>(&'q mut self, world: &'q mut World) -> PreparedQueryMap<'q, Q> {
```

Comment:
I am not particularly fond of this name and would be glad for a suggestion for a better one.

Reply:
What if we globally replaced "query map" with "view"? It's more concise, and IMO at least as clear.

Reply:
Renamed it, it is called `struct View` now and this method became `view_mut`.